src/objects/js/components/admin/permissions/permission-form.js

```diff
@@ -65,6 +65,7 @@ const PermissionForm = ({objectFields, tokenChoices, objecttypeChoices, modeChoi
                 id="id_object_type"
                 label="Object type:"
                 choices={objecttypeChoices}
+                helpText="Changing the object_type will not maintain the previously selected authorization fields."
```

Comment:
Let's use the name as it is displayed in the admin
```suggestion
                helpText="Changing the Object type will not maintain the previously selected authorization fields."
```